plugins/modules/system/aix_filesystem.py

```diff
@@ -170,7 +170,7 @@
 '''
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible_collections.ansible.posix.plugins.module_utils.ismount import ismount
+from ansible_collections.ansible.posix.plugins.module_utils.mount import ismount
```

Comment:
Hmm, this now depends on which version of the ansible.posix collection is installed. How about changing this to:
```suggestion
try:
    from ansible_collections.ansible.posix.plugins.module_utils.mount import ismount
except ImportError:
    from ansible_collections.ansible.posix.plugins.module_utils.ismount import ismount
```
That way it will also work with older versions of the ansible.posix collection.

Reply:
As discussed on IRC, let's not do this but bump the requirement in galaxy.yml to 1.0.0.